test_schemas/en/test_new_routing_number_less_than_or_equal_single_condition.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
The `_single_condition` bit should be dropped from the schema names, and the references need to be updated.